### PR TITLE
chore: change report time filter values

### PR DESCRIPTION
**Pull request by cbernat**
Changed time filter options in reports. 

image

Issue: DBR-184

### CODE REVIEW

File: src/i18n/es.json [hunk 1]
```diff
@@ -65,10 +65,9 @@
     "domain_not_verified": "Dominio no verificado",
     "pages": "Páginas",
     "rank_time": "Rango de tiempo",
-    "rank_time_item1": "última semana",
-    "rank_time_item2": "último mes",
-    "rank_time_item3": "últimos 3 meses",
-    "rank_time_item4": "últimos 6 meses",
+    "rank_time_item1": "1 semana",
+    "rank_time_item2": "2 semanas",
+    "rank_time_item3": "3 semanas",
```
Comment (andresmoschini) [inline src/i18n/es.json]: What do you think about renaming it as something related to actual values?

Comment (cbernat) [reply src/i18n/es.json]: If you think it's best... I don't know how sure are we of these values...

Comment (andresmoschini) [reply src/i18n/es.json]: It is not really necessary, but what do you think about using pluralization features?

**Comment by andresmoschini** [comment]
👍

**Pull request merged**